### PR TITLE
Update and refactor docs

### PR DESCRIPTION
**Pull request by pacospace**
## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,129 +27,54 @@ to Kafka for each package release to be solved using Thoth [Solver](https://gith
...
+Consumer is currently able to handle the following Kafka messages focused on:
 
-#### UnresolvedPackageMessage
+### Increase Thoth Knowledge
 
-- [UnresolvedPackageMessage](https://github.com/thoth-station/messaging/blob/master/thoth/messaging/unresolved_package.py).
+The following messages are sent by different Thoth components:
 
-This message received contains:
+- [PackageReleasedMessage](https://github.com/thoth-station/investigator/blob/master/iinvestigator/investigator/package_released/README.md).
```
Comment (saisankargochhayat) [inline README.md:36]: r(iinvestigator, investigator)

Comment (pacospace) [reply README.md:36]: nice catch!

**Review by saisankargochhayat — APPROVED** [review]
LGTM! :100: Thanks!

**Pull request merged**